lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -57,6 +57,12 @@
     type: bool
     default: 'yes'
     version_added: "2.1"
+  no_create_db:
+    description:
+      - Option used to exclude CREATE DATABASE and USE commands
```

Comment:
```suggestion
      - Option used to exclude CREATE DATABASE and USE commands. Used only with I(state=export).
```
Please, determine explicitly with what this option can be used (i'm not sure that it is only state=export, needs to be rechecked)